### PR TITLE
OHRM5X-271 : Test script for Skills screen

### PR DESCRIPTION
**Pull request by ShenaliFernando**

### CODE REVIEW

**Comment by samanthajayasinghe** [comment]
@ShenaliFernando There are some conflicts in this PR

File: symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js [hunk 1]
```diff
@@ -1,116 +1,166 @@
+<<<<<<< e0e330c4cc79b7dc0e369215f90e5aad5e705f8a
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js:1]: resolve this conflict

File: symfony/test/functional/cypress/fixtures/admin.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "admin": {
+        "userName": "Admin",
+        "password": "admin@OHRM123",
+        "fullName": "Jacqueline White"
+    }
+}
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress/fixtures/admin.json:7]: add new line

File: symfony/test/functional/cypress.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "http://php80/orangehrm/web/index.php",
+    "baseUrl": "http://php80/orangehrm/symfony/web/index.php",
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress.json:2]: revert this change

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: symfony/test/functional/cypress/fixtures/admin.json (hunk 2 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/fixtures/admin.json:7]: New line added

File: symfony/test/functional/cypress.json (hunk 3 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress.json]: Done

**Review by samanthajayasinghe — APPROVED** [review]

**Pull request merged**